Gemfile

```diff
@@ -0,0 +1,4 @@
+source "https://rubygems.org"
+
+gem "cocoapods", "~> 1.5.3"
+gem "fastlane", "~> 2.106.0"
```

Comment:
Why does it need fastlane?

Reply:
Cool, just asking 🙃

Reply:
@kafejo So now it is enough to run `bundle update` and `bundle exec fastlane update_pod next_version:"4.0.0"`.